data/mods/Magiclysm/Spells/monsterspells.json

```diff
@@ -16,5 +16,44 @@
     "range_increment": 0.2,
     "effect": "projectile_attack",
     "extra_effects": [ { "id": "light_healing", "hit_self": true } ]
+  },
+  {
+    "id": "rocket_punch",
+    "type": "SPELL",
+    "name": "Rocket Punch",
+    "description": "Ejects giant fist from arm.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "NO_LEGS" ],
+    "damage_type": "bash",
+    "min_damage": 30,
+    "max_damage": 40,
+    "damage_increment": 2,
+    "max_level": 5,
+    "min_range": 1,
+    "max_range": 2.5,
+    "range_increment": 0.3,
+    "effect": "projectile_attack"
+  },
+  {
+    "id": "gas_attack",
+    "type": "SPELL",
+    "name": "Gas Attack",
+    "description": "Spreads toxic gas around itself.",
+    "valid_targets": [ "self" ],
+    "flags": [ "NO_HANDS", "NO_LEGS" ],
+    "damage_type": "bio",
+    "min_damage": 1,
+    "max_damage": 1,
+    "max_level": 5,
+    "min_range": 1,
+    "max_range": 3,
+    "range_increment": 0.2,
+    "effect": "target_attack",
+    "field_id": "fd_toxic_gas",
+    "field_chance": 100,
```

Comment:
this spell confuses me: so you want a 1 in 100 chance of spawning  gas cloud on the tile the caster is standing on? what's the range for then? if you want to be able to spawn gas clouds on the ground you need to have `ground` as a valid target. additionally with an AOE of 0 it'll only affect the target tile (not necessarily a problem if that's the goal)